src/components/ActionCard.tsx

```diff
@@ -92,23 +52,44 @@ const ActionCard: React.FC<IProps> = ({
   const isImageURL = typeof image === "string"
 
   return (
-    <Card to={to} className={className} hideArrow={true}>
-      <ImageWrapper
-        isRight={isRight}
-        isBottom={isBottom}
+    <Link
```

Comment:
Instead of wrapping this component with a Link, lets use the `LinkOverlay` component that Chakra offers for this specific scenario https://chakra-ui.com/docs/components/link-overlay/usage

I did a similar implementation in https://github.com/ethereum/ethereum-org-website/pull/7721

Reply:
Ahh, I see. So, if I understand the usage correctly, the whole card will be wrapped in a LinkBox and then, I can just wrap the heading in a LinkOverlay
And the LinkBox will elevate this link in a way that to the user it looks like the whole card is a link.